projects/saturn/src/main/java/io/fairspace/saturn/services/metadata/MetadataApp.java

```diff
@@ -4,22 +4,33 @@
 import io.fairspace.saturn.services.BaseApp;
 import io.fairspace.saturn.services.PayloadParsingException;
 import io.fairspace.saturn.services.metadata.validation.ValidationException;
-import lombok.extern.log4j.*;
+import lombok.extern.log4j.Log4j2;
 import org.apache.jena.rdf.model.Model;
 import spark.Request;
 
 import static io.fairspace.saturn.services.errors.ErrorHelper.errorBody;
 import static io.fairspace.saturn.services.errors.ErrorHelper.exceptionHandler;
-import static io.fairspace.saturn.services.metadata.Serialization.*;
+import static io.fairspace.saturn.services.metadata.Serialization.deserialize;
+import static io.fairspace.saturn.services.metadata.Serialization.getFormat;
+import static io.fairspace.saturn.services.metadata.Serialization.serialize;
 import static io.fairspace.saturn.util.ValidationUtils.validate;
 import static io.fairspace.saturn.util.ValidationUtils.validateIRI;
-import static javax.servlet.http.HttpServletResponse.*;
+import static java.lang.Boolean.TRUE;
+import static javax.servlet.http.HttpServletResponse.SC_BAD_REQUEST;
+import static javax.servlet.http.HttpServletResponse.SC_FORBIDDEN;
+import static javax.servlet.http.HttpServletResponse.SC_NO_CONTENT;
 import static org.apache.jena.rdf.model.ResourceFactory.createResource;
 import static org.eclipse.jetty.http.MimeTypes.Type.APPLICATION_JSON;
-import static spark.Spark.*;
+import static spark.Spark.delete;
+import static spark.Spark.get;
+import static spark.Spark.patch;
+import static spark.Spark.put;
 
 @Log4j2
 public class MetadataApp extends BaseApp {
+
+    private static final String DO_VIEWS_UPDATE = "doViewsUpdate";
```

Comment:
Please document it in our documentation.

Reply:
good catch, done